### DISCUSSION TITLE
can you give example full code about MVP

### DISCUSSION THREAD

**Discussion by bobwatcherx** [iron3oxide/fletched#30] [Q&A]
I have read your mvp documentation. but I don't understand and I copy and paste it into my code. instead a datatable appears that I don't know the source of. so can you give a simple example with your code about mvp

**Comment by bobwatcherx** [comment]
and also routedApp

**Comment by iron3oxide** [comment]
This is definitely on my ToDo-List, but if I'm being honest, I probably won't get to it in the near future. 
I'm very busy with university and other stuff right now and I'm also increasingly unsure if the upstream project (Flet) will fully suit my needs in the foreseeable future. This project is therefore a bit dormant at the moment; I'll keep merging the dependabot PRs and fix issues when they are opened, but will not develop new features for now.
